### ISSUE TITLE
no-unused-class for class names with variables in it

### ISSUE THREAD

**Issue by nelsonleite** [atfzl/eslint-plugin-css-modules#17]
If I have a class like `myClass-${myVar}`, it throws a false `no-unused-class`.

Let's say I have this markup:

```jsx
const myVar = 'container';

<div className={ `myClass-${myVar}` }>
    This container is blue
</div>
```

with these styles:

```css
.myClass-container {
    color: blue;
}
```

The linter does not recognise `myClass-${myVar}` as `myClass-container`.

**Comment by TrySound** [comment]
@nelsonleite It's not css-modules. It's just a template literal with interpolation. Nothing to lint.

**Comment by nelsonleite** [comment]
But I wonder: shouldn't it be able to recognise as a valid class? Otherwise I just have to disable the linter...

**Comment by TrySound** [comment]
Show config

**Comment by atfzl** [comment]
https://en.wikipedia.org/wiki/Static_program_analysis
> Static program analysis is the analysis of computer software that is performed without actually -executing programs
